src/main/java/org/synyx/urlaubsverwaltung/person/PersonServiceImpl.java

```diff
@@ -60,6 +60,8 @@ public Person create(String username, String lastName, String firstName, String
         accountInteractionService.createDefaultAccount(person);
         workingTimeService.createDefaultWorkingTime(person);
 
+        applicationEventPublisher.publishEvent(new PersonCreatedEvent(this, persistedPerson.getId()));
```

Comment:
vs. `create(Person person)`? Eventuell wollen beide Methoden das selbe Verhalten implementieren

Reply:
Bei der anderen Methode war ich unsicher warum es die überhaupt gibt. Wird nämlich nur in Tests verwendet. Schaue ich mir nochmal an.

Reply:
hatte vor kurzem das Gleiche Erlebnis. Kann schon sein das es ein paar Methoden ohne produktiven Einfluss gibt.

Reply:
Ansatz mit den ApplicationEvents habe ich komplett verworfen (siehe Beschreibung des PRs)